### ISSUE TITLE
Requester signup: no DigiD connection? No possible way to apply anymore

### ISSUE THREAD

**Issue by GerbenBosschieter** [teamforus/general#354]
Learn more about change requests here: https://bit.ly/39CWeEE

### Requested by:
- @GerbenBosschieter 

### Change description
If there is no active connection with DigiD the user will be confronted with an error. (no problem with the error).
If the user wants to back to the beginning page it will be redirected to the homepage while logged in, but he didn't finish the application. 

The user wants to finish the application but after getting redirected while logging in, there is no possibility anymore.

**Note**
- This has also to do with removing the **Start aanvraag** button from the header content area. because of the new navigation.

<img width="936" alt="Screen Shot 2020-08-12 at 11 17 52" src="https://user-images.githubusercontent.com/38419514/89998349-b564c100-dc8d-11ea-9804-5dd7bfd2e79c.png">

**Comment by maxvisser** [comment]
Yeah probably we need the aanvragen page. Would be good to redirect people back to this page. @GerbenBosschieter this could also be a potential issue for Nijmegen if ever the DigiD connection is down
